### ISSUE TITLE
4.1 Release Tasks

### ISSUE THREAD

**Issue by jimmyDunne** [opensim-org/opensim-core#2537]
- [x] Double check that all Matlab Tests Pass locally
- [x] Document Python 3 update on Confluence. Use vanilla Python as the default version for instructions. Add note about building from source for 2.7
- [x] Update splash screen
- [x] Update GUI doc landing page
- [x] Write what's new section on confluence
- [x] Other confluence/user guide updates
- [x] Post new doxygen
- [x] Add new html files for tutorials

**Comment by aymanhab** [comment]
Tutorials live in a different repo opensim-models at
https://github.com/opensim-org/opensim-models/tree/master/Tutorials/doc

**Comment by jenhicks** [comment]
No wonder I couldn't find them in the GUI repo ;). Do you have the files you need then?

**Comment by jenhicks** [comment]
Ah ... I see that they are html docs and not pdfs. Is it possible to use PDFs instead? This might be less error prone and would result in fewer files in the repo.

**Comment by aymanhab** [comment]
We used pdf for a while but changed to html for a couple reasons:
- Online version in confluence is html, we don't need to freeze it into pdf that's hard to compare/diff/modify
- In sync. with user guide, help and other links all using html, we just switch url based on whether online/offline.
- Not sure all browsers handle pdf the same on windows/mac, while html is better supported.
We can change to pdf if you think it's easier to maintain but would be good to go back to see why we switched in the first place.
